scripts/docker/ubuntu/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM ubuntu:18.04
-RUN apt-get update -y
-RUN apt-get install -y git 
+RUN apt-get update -y && apt-get install -y git
 ENV PASH_HOST="docker"
+ENV PASH_TOP=/opt/pash
 # download PaSh
-RUN git clone https://github.com/binpash/pash.git /opt/pash --depth 1
-# install PaSh and setup shell paths
-RUN yes | bash /opt/pash/scripts/install.sh -p
+RUN git clone https://github.com/dkarnikis/pash.git /opt/pash --depth 1 -b pash-setup
```

Comment:
@dkarnikis Should we be pulling from this repo instead of binpash main here?

Reply:
oops. you are absolutely correct.